### PR TITLE
Use colors for logging.

### PR DESCRIPTION
**Pull request by kyrofa**
This fixes LP: [#1572186](https://bugs.launchpad.net/snapcraft/+bug/1572186) by using the following colors:

  - INFO: Green
  - WARNING: Yellow
  - ERROR: Dark red
  - CRITICAL: Light red

### CODE REVIEW

**Comment by kyrofa** [comment]
retest this please

File: integration_tests/__init__.py [hunk 1]
```diff
@@ -93,17 +93,9 @@ def login(self, email='u1test+snapcraft@canonical.com',
...
         result = 'successful' if expect_success else 'failed'
-        process.expect_exact(
-            # bold.
-            '\x1b[1m'
-            'Login {}.'.format(result) +
-            '\x1b[0m\r\n')
+        process.expect_exact('Login {}.'.format(result))
 
     def logout(self):
         output = self.run_snapcraft('logout')
```
Comment (elopio) [inline integration_tests/__init__.py]: This is a lot better, thanks.

File: snapcraft/internal/log.py [hunk 2]
```diff
@@ -37,6 +33,25 @@ def filter(self, record):
...
 
 
+class _ColoredFormatter(logging.Formatter):
+    reset = '\033[0m'
+    level_colors = {
+        'INFO': '\033[0;32m',  # Green
+        'WARNING': '\033[1;33m',  # Yellow
+        'ERROR': '\033[0;31m',  # Dark red
+        'CRITICAL': '\033[1;31m',  # Light red
+    }
```
Comment (elopio) [inline snapcraft/internal/log.py]: These are constants. You should use upper case letters in the var name.

**Comment by elopio** [comment]
:+1:
just a complaint about the var names of the constants.

File: snapcraft/internal/log.py (hunk 2 above)
Comment (kyrofa) [inline snapcraft/internal/log.py]: Done.

**Pull request merged**